cranelift/codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1671,6 +1671,7 @@ fn define_simd(
     let uload32x2 = shared.by_name("uload32x2");
     let uload32x2_complex = shared.by_name("uload32x2_complex");
     let snarrow = shared.by_name("snarrow");
+    let unarrow = shared.by_name("unarrow");
```

Comment:
Nit in this commit message (the one introducing unarrow): `unsigned`, not `signed`